tests/Unit/Domain/Services/TalkRating/OneToTenRatingTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Unit\Domain\Services\TalkRating;
+
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use OpenCFP\Domain\Model\TalkMeta;
+use OpenCFP\Domain\Services\Authentication;
+use OpenCFP\Domain\Services\TalkRating\OneToTenRating;
+
+/**
+ * @covers \OpenCFP\Domain\Services\TalkRating\OneToTenRating
+ */
+class OneToTenRatingTest extends \PHPUnit\Framework\TestCase
+{
+    use MockeryPHPUnitIntegration;
+
+    /**
+     * @dataProvider validRatingProvider
+     */
+    public function testGoodRatingsAreSuccessful($rating)
+    {
+        $mockAuth    = Mockery::mock(Authentication::class)->shouldIgnoreMissing();
+        $metaMock    = Mockery::mock(TalkMeta::class);
+        $oneToTen    = new OneToTenRating($metaMock, $mockAuth);
```

Comment:
Can we reduce the whitespace here?

Alternatively, we can just wait for approval of #820! 🤓